src/openrct2/world/Scenery.h

```diff
@@ -181,12 +182,13 @@ enum
     PATH_BIT_FLAG_IS_QUEUE_SCREEN = 1 << 8
 };
 
-enum
+enum class PathBitDrawType : uint8_t
 {
-    PATH_BIT_DRAW_TYPE_LIGHTS,
-    PATH_BIT_DRAW_TYPE_BINS,
-    PATH_BIT_DRAW_TYPE_BENCHES,
-    PATH_BIT_DRAW_TYPE_JUMPING_FOUNTAINS
+    Light,
+    Bin,
+    Bench,
+    JumpingFountain
+
```

Comment:
Could you delete this white space and end the fountain entry with a comma for the unlikely event that we are adding more to the enum

Reply:
Done